README.md

```diff
@@ -52,35 +58,15 @@ Then login as NOMIS_BATCHLOAD user and go to ‘manage locations’ and then 'AB
 - `npm run clean` -> cleans previously generated files
 - `npm run build` -> cleans and regenerates assets.
 - `npm run typecheck` -> runs typescript type checking
+- `npm run start:feature` -> runs the application configured to run integration tests
 
-# Environment variables
-
-The following environment variables are used and values should be supplied for correct operation but have defaults.
-The easiest way to supply them is to create a .env file in the project root.
 
-- NOMIS_API_URL - url for nomis elite2 api entry point eg http://localhost:8080/elite2api
-- NOMIS_AUTH_URL - url for nomis oauth server eg http://localhost:9090/auth
-- DOMAIN - domain where the app is running eg http://localhost:3000
-
-Oauth config:
-
-- API_CLIENT_SECRET
```

Comment:
removed this partial/outdated list and suggesting using feature.env as reference